ca_description/urdf/create_base.xacro

```diff
@@ -44,6 +45,12 @@
       </collision>
     </link>
 
+    <xacro:property name="diameter" value="0.3485"/>
+    
+    <!-- Introducing color sensors in each side -->
+    <xacro:color_sensor prefix="left" side="${diameter}"/>
```

Comment:
Imagine that want to use this sensor in another part of the robot. What are you gonna do?

You can't. So, Instead of passing `side`, insert an origin block.